ClearKeep/Scenes/ChatGroup/Views/GroupChatContentView.swift

```diff
@@ -0,0 +1,242 @@
+//
+//  GroupChatContentView.swift
+//  ClearKeep
+//
+//  Created by đông on 28/03/2022.
+//
+
+import SwiftUI
+import Combine
+import Common
+import CommonUI
+
+private enum Constant {
+	static let spacerTopView = 90.0
+	static let spacerBottomView = 20.0
+	static let spacer = 25.0
+	static let spacerBottom = 45.0
+	static let paddingVertical = 14.0
+	static let paddingHorizontal = 24.0
+	static let heightButton = 40.0
+	static let cornerRadiusButtonNext = 40.0
+	static let cornerRadiusTagUser = 80.0
+	static let sizeImage = 64.0
+	static let paddingButtonNext = 60.0
+
+}
+
+struct GroupChatContentView: View {
+	// MARK: - Variables
+	let inspection = ViewInspector<Self>()
+	@Environment(\.injected) private var injected: DIContainer
+	@Environment(\.colorScheme) private var colorScheme
+	@Environment(\.presentationMode) private var presentationMode: Binding<PresentationMode>
+	@State private(set) var samples: Loadable<[IGroupChatModel]>
+	@State private(set) var searchText: String
+	@State private(set) var searchStyle: TextInputStyle = .default
+	@State private(set) var isShowingView: Bool = false
+	@State private(set) var isSelectedUser: Bool = false
+
+	// MARK: - Init
+	public init(samples: Loadable<[IGroupChatModel]> = .notRequested,
+				searchText: String = "",
+				inputStyle: TextInputStyle = .default) {
+		self._samples = .init(initialValue: samples)
+		self._searchText = .init(initialValue: searchText)
+	}
+
+	// MARK: - Body
+	var body: some View {
+		content
+			.onReceive(inspection.notice) { inspection.visit(self, $0) }
+		//			.background(background)
+			.edgesIgnoringSafeArea(.all)
+			.navigationBarBackButtonHidden(true)
+	}
+}
+
+// MARK: - Private
+private extension GroupChatContentView {
+	var content: AnyView {
+		AnyView(notRequestedView)
+	}
+}
+
+// MARK: - Loading Content
+private extension GroupChatContentView {
+	var notRequestedView: some View {
+		VStack {
+			buttonBackView
+				.padding(.top, Constant.spacerTopView)
+			searchInput.padding(.top, Constant.paddingVertical)
+			tagUsers
+			addAnotherUserButton
+			if isShowingView {
+				CommonTextField(text: $searchText,
+								inputStyle: $searchStyle,
+								placeHolder: "GroupChat.User.Add.Placeholder".localized,
+								keyboardType: .default,
+								onEditingChanged: { isEditing in
+					if isEditing {
+						searchStyle = .default
+					} else {
+						searchStyle = .highlighted
+					}
+				})
+			}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/3249a8332ccef754f5404bc6377103b48f3d2d68/ClearKeep/Scenes/ChatGroup/Views/GroupChatContentView.swift#L86"></span>Else and catch should be on the same line, one space after the previous declaration.<br />
<code>statement_position</code> <code>GroupChatContentView.swift:86</code></td>
    </tr>
  </tbody>
</table>